Build/Projects/FrameworkReferences.definition

```diff
@@ -25,6 +29,7 @@
         Name="OpenTK"
         Path="ThirdParty/Dependencies/OpenTK.dll" />
     </Service>
+    <Reference Include="MonoGame.Framework.Core.Include" />
```

Comment:
Ok, there are two types of references used here:
```
MonoGame.Framework.Core
```
Which is the reference to the Core PCL, containing those non-platform specific classes
```
MonoGame.Framework.Core.include
```
which includes the non-platform specific files in the PCL in the host project.  Meaning this project doesn't use the PCL.  This is done for Linux and Web which don't support PCL references as yet.  I've also excluded WindowsPhone8 from the PCL as it caused a mess with the XNA references and there's no need to change a platform that will be phased out soon.

@cra0zy has indicated that Linux does support PCL references but to keep it clean, I've not included it in this change.  Will submit a separate one once this is done.